File: Fundamentals C#/Basic Syntax, Cond State, Loops/Basic Syntax, Conditional Statements and Loops - Lab/07. Theatre Promotions.cs
using System;

namespace _7._Theatre_Promotions
{
    internal class Program
    {
        static void Main(string[] args)
        {
            string day = Console.ReadLine();
            int age = int.Parse(Console.ReadLine());            
            double price = 0;
                       

            if (age >= 0 && age <= 18)
            {
                if (day == "Weekday")
                {
                    price = 12;
                }
                if (day == "Weekend")
                {
                    price = 15;
                }
                if (day == "Holiday")
                {
                    price = 5;
                }
            }
            if (age > 18 && age <= 64)
            {
                if (day == "Weekday")
                {
                    price = 18;
                }
                if (day == "Weekend")
                {
                    price = 20;
                }
                if (day == "Holiday")
                {
                    price = 12;
                }
            }
            if (age > 64 && age <= 122)
            {
                if (day == "Weekday")
                {
                    price = 12;
                }
                if (day == "Weekend")
                {
                    price = 15;
                }
                if (day == "Holiday")
                {
                    price = 10;
                }
            }
            if (age < 0 || age > 122)
            { 
                Console.WriteLine("Error!");                
            }
            if(price>0)
                Console.WriteLine($"{price}$");

        }
    }
}

File: JavaScript Front-End/01. Syntax Fundamentals/Syntax Fundamentals Lab/07. Theatre Promotions.js
function solve(day, age) {
    let price;

    if (day == 'Weekday') {
        if (age >= 0 && age <= 18) {
            price = 12;
        } else if (age > 18 && age <= 64) {
            price = 18;
        } else if (age > 64 && age <= 122) {
            price = 12;
        } else {
            console.log('Error!');
            return;
        }
    }
    else if (day == 'Weekend') {

        if (age >= 0 && age <= 18) {
            price = 15;
        } else if (age > 18 && age <= 64) {
            price = 20;
        } else if (age > 64 && age <= 122) {
            price = 15;
        } else {
            console.log('Error!');
            return;
        }
    } else {
        if (age >= 0 && age <= 18) {
            price = 5;
        } else if (age > 18 && age <= 64) {
            price = 12;
        } else if (age > 64 && age <= 122) {
            price = 10;
        } else {
            console.log('Error!');
            return;
        }
    }
    console.log(`${price}$`);
}

solve('Holiday', -1)